airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/StreamCopier.java

```diff
@@ -18,6 +18,13 @@
    */
   void write(UUID id, AirbyteRecordMessage recordMessage, String fileName) throws Exception;
 
+  /**
+   * Closes the writer for the stream to the current staging file. The staging file must be of a
```

Comment:
aaahhh I think I understand - is this saying that after calling this method, _only_ the current staging file will have an open stream? (if yes, then I think `closeNonCurrentStagingFileWriters` would be a better name. Sorry for the confusion!)